resources/assets/components/CampaignSingle/index.js

```diff
@@ -34,9 +23,48 @@ class CampaignSingle extends React.Component {
     this.filterPosts = this.filterPosts.bind(this);
   }
 
+  componentDidMount() {
+    this.api.get('api/v2/posts', {
+      filter: {
+        status: 'accepted',
+        campaign_id: this.props.campaign.id,
+      },
+      include: 'signup,siblings',
```

Comment:
Let's dig into this together sometime this week (and definitely doesn't block merging).